src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/HttpClient/SendAsync.cs

```diff
@@ -95,6 +127,9 @@ private static Uri TryGetAbsoluteUri(HttpRequestMessage httpRequestMessage, Syst
             if (httpRequestMessage.RequestUri?.IsAbsoluteUri == true)
                 return httpRequestMessage.RequestUri;
 
+            if (httpClient == null)
```

Comment:
Did we test to see what happens when you instantiate an `HttpClient` with a base address and then make a call to a relative URI?

Reply:
Since we instrument `SocketsHttpHandler` methods in .Net 5 to cover every `HttpClient` call paths, we could only get the Uri from the `HttpRequestMessage`. Luckily, my test shows that the absolute uri is already available by the time the SocketsHttpHandler is running. I have tested the following test case:

```
           HttpClient client = new();
            client.BaseAddress = new Uri("https://localhost:5001");
            Forecast[] forecasts = await client.GetFromJsonAsync<Forecast[]>("WeatherForecast");
```

Reply:
Ok, let's get this test case incorporated into your integration tests that you plan to bring in.